### PR TITLE
First Design of the Header

### PR DESCRIPTION
**Pull request by ChristianSeelemann**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,8 +13,12 @@
...
   <body>
     <header>
-      <div class="logo">Logo</div>
-      <div class="content">Content</div>
+      <div class="logo">Chris</div>
+      <div class="content">
+        <a href="#hero"><li>About me</li></a>
+        <a href="#skills"><li>Skills</li></a>
+        <a href="#contact"><li>Contact</li></a>
+      </div>
```
Comment (marwinburesch) [inline index.html]: A semantic element would be perfect here! :)
https://dev.to/kenbellows/stop-using-so-many-divs-an-intro-to-semantic-html-3i9i?utm_source=mybridge&utm_medium=blog&utm_campaign=read_more

```suggestion
      <nav class="content">
        <a href="#hero"><li>About me</li></a>
        <a href="#skills"><li>Skills</li></a>
        <a href="#contact"><li>Contact</li></a>
      </nav>
```

**Review by marwinburesch — CHANGES_REQUESTED** [review]
Nice selector game :) 
Maybe some empty lines between ruleset declarations?

File: index.html [hunk 2]
```diff
@@ -13,8 +13,12 @@
   </head>
   <body>
     <header>
-      <div class="logo">Logo</div>
-      <div class="content">Content</div>
+      <div class="logo">Chris</div>
+      <div class="content">
+        <a href="#hero"><li>About me</li></a>
+        <a href="#skills"><li>Skills</li></a>
+        <a href="#contact"><li>Contact</li></a>
```
Comment (LyriCaGitHubber) [inline index.html:20]: `<ul> <li> <a href="link" </li> </ul>` that would be my resolve. :-)

**Review by LyriCaGitHubber — APPROVED** [review]
Top!

**Pull request merged**